packages/sdk/src/index.ts

```diff
@@ -129,5 +207,14 @@ export function Sdk() {
       const result = await scheduleXcmpTaskWithPayThroughRemoteDerivativeAccountFlow(params);
       return result;
     },
+    /**
+     * Schedule XCMP price task with PayThroughRemoteDerivativeAccount instruction sequances
+     * @param params Operation params
+     * @returns 
+     */
+    scheduleXcmpPriceTaskWithPayThroughRemoteDerivativeAccountFlow: async (params: ScheduleXcmpPriceTaskWithPayThroughRemoteDerivativeAccountFlowParams): Promise<SendExtrinsicResult> => {
```

Comment:
What the point is this function? If there’s only one line in a function, it’s constructed wrong.

Reply:
Fixed

Reply:
uhh there’s no new code commit. Did you forget to push the changes to this PR? 😅

Reply:
Yes, I forgot to push the changes.